tests/cpu_bugs/kvm_guest_mitigations.pm

```diff
@@ -0,0 +1,380 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# KVM Guest mitigations test include mitigations=auto/off/auto,nosmt
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: Qi Wang <qwang@suse.com>
+package kvm_guest_mitigations;
+use strict;
+use warnings;
+use base "consoletest";
+use bootloader_setup;
+use Mitigation;
+use ipmi_backend_utils;
+use power_action_utils 'power_action';
+use testapi;
+use utils;
+use bmwqemu;
+use Data::Dumper;
+use Mitigation;
+our $CPU_MODE = get_required_var("CPUMODE");
+my $qa_password = get_var("QA_PASSWORD", "nots3cr3t");
+my $install_media_url = get_required_var('INSTALL_MEDIA');
+my $hostmitigation = get_required_var('HOSTMITIGATION');
+my $guest_name = get_var('GUEST_NAME');
+my $extra_guest_kernel_param = get_required_var('EXTRA_KERNEL_PARAMETER_FOR_GUEST');
+if (!$guest_name) {
+    $guest_name = "kvm_" . "$CPU_MODE" . "_guest";
+}
+my $guest_imagepool_path = "/kvm_" . "$CPU_MODE";
+my $guest_ip_addr;
+
+#Skylake configuration
+my $mitigations_auto_on_skylake_passthrough = {"mitigations=auto" => {
```

Comment:
I think the cases is not enough, should contain spectre_v2 and its sub cases at least

Reply:
Additionally, you should do some test for xen guest, because you changed /lib/mitigations.pm file which is invoked by xen test

Reply:
> I think the cases is not enough, should contain spectre_v2 and its sub cases at least

Refer to http://10.67.129.4/tests/59088#  Testing only include mitigations,so I didn't add other.

Reply:
> ctre_v2 and its sub cases at least

I will create a task on progess for enhance this.https://progress.opensuse.org/issues/139025

Reply:
For currently,I think it's ok,later when we have more requirements,we can enhance it.

Reply:
> > Additionally, you should do some test for xen guest, because you changed /lib/mitigations.pm file which is invoked by xen test
> 
> OK, I will do dome test for xen.

http://10.67.129.7/tests/2543#
http://10.67.129.7/tests/2541